Add upgradeability script and jobs - final phase (#518)
kyma-project/test-infra: prow/scripts/cluster-integration/README.md

### Project structure

The structure of the folder looks as follows:

``` 

> ├── helpers # This directory contains helpers scripts used by pipeline jobs.

Reviewer: ```suggestion
├── helpers # This directory contains helper scripts used by pipeline jobs.
```

> ├── kyma-gke-integration.sh # The purpose of this script is to install and test Kyma on real GKE cluster.

Reviewer: ```suggestion
├── kyma-gke-integration.sh # This script installs and tests Kyma on a real GKE cluster.
```

> ├── kyma-gke-nightly.sh # The purpose of this script is to create from master branch a long-lived GKE cluster. This cluster should be recreated once per day. 

Reviewer: ```suggestion
├── kyma-gke-nightly.sh # This script creates a long-lived GKE cluster from the master branch. This cluster should be recreated once a day. 
```

> └── kyma-gke-upgrade.sh # The purpose of this script is to install last Kyma release on GKE cluster, upgrade it with current changes from master/PR/release branch and trigger Kyma testing script.

Reviewer: ```suggestion
└── kyma-gke-upgrade.sh # This script installs the last Kyma release on a GKE cluster and upgrades it with current changes from the PR, master, or release branch. It also triggers the Kyma testing script.
```
